examples/layer-browser/src/examples/experimental-layers.js

```diff
@@ -1,7 +1,7 @@
 import {MeshLayer, PathOutlineLayer, PathMarkerLayer, AdvancedTextLayer} from 'deck.gl-layers';
 
 // TODO: remove hard path once deck.gl-layers published with GPUScreenGridLayer
-import GPUScreenGridLayer from '../../../../src/experimental-layers/src/gpu-screen-grid-layer/gpu-screen-grid-layer';
+import GPUScreenGridLayer from '@deck.gl/experimental-layers/gpu-screen-grid-layer/gpu-screen-grid-layer';
```

Comment:
Why are we importing it with a long path like this and not as `import {GPUScreenGridLayer} from '@deck.gl/experimental-layers;`

The whole point of `experimental-layers` is to export "experimental layers".